### ISSUE TITLE
I just wrote a working plugin for Atom: Linter-Ruby-Reek = LRR

### ISSUE THREAD

**Issue by ahmadseleem** [troessner/reek#603]
Hey,

I really liked your work and wanted to use it in Atom editor.
The other package was broken on your side and Linter API side and seemed inactive.
I couldn't wait.

I call it: LRR 

https://atom.io/packages/linter-ruby-reek

I hope that's a little contribution and hopefully I can do more...

Thanks!

**Comment by mvz** [comment]
Hi @ahmadseleem, perhaps you can contribute your fixes back to [linter-reek](https://github.com/AtomLinter/linter-reek). It seems you found fixes for the [issues reported there](https://github.com/AtomLinter/linter-reek/issues). I really don't think its development is inactive.

**Comment by ahmadseleem** [comment]
**It wasn't an issue. it was not written to target reek or Linter API at all.**:

I did send the developer two messages in the issues and I got no answer then I deleted my comments when I found things I mentioned below.

Before, the maintainer just wrote (https://github.com/AtomLinter/linter-reek/issues/10) then, I commented on that issue, too.
Before that comment, I was already working on the plugin. I just didn't have time to publish it until yesterday.
 
But, I looked into the code and it needed a complete rewrite.
It wasn't an issue. it was not written to target reek or Linter API at all. Not even the Linter's API template:
https://github.com/atom-community/linter/wiki/Migrating-to-the-new-API

Go ahead check the code and you'll notice that! no logic was added over the Linter template.
https://github.com/AtomLinter/linter-reek

I was *Disappointed* because, how can I *Publish* something and say it's working and not targeting anything or *never* worked from start, while I know that!

Not only it was just copying the template, Portions from the Linter template were deleted, I don't know why?!

I'm not making something out of that or adding any button for donations... and it's MIT Licensed. very simple.
Thanks for mentioning that! and thanks for your work!
